src/naksu/mebroutines/install/install.go

```diff
@@ -54,17 +67,18 @@ func newServer(boxType string, imageURL string, versionURL string) error {
 		if removeErr != nil {
 			log.Debug("Failed to remove image file %s: %v", mebroutines.GetImagePath(), removeErr)
 		}
-
+		progress.CloseProgressDialog(progressDialog)
 		return fmt.Errorf("failed to create new vm: %v", err)
 	}
 
-	progress.TranslateAndSetMessage("Removing temporary raw image file")
+	updateProg("Removing temporary raw image file", 100*(3/3))
```

Comment:
CheckStyle (gocritic) valittaa tästä: "dupSubExpr: suspicious identical LHS and RHS for `/` operator". Eli ilmeisesti se ei tykkää tuosta `3/3`-subexpressionista. Tykkään tästä selkeästä `100*(2/3)`-tyylistä, mutta ehkäpä nämä voisi silti kirjoittaa auki gocriticin lepyttämiseksi: `33`, `67` ja `100`.